index.js

```diff
@@ -0,0 +1,38 @@
+(function() {
+
+  const Actuate = animations => $ => new Promise ((resolve, reject) => {
+
+    const commands = Array.isArray(animations) ?
+      animations : animations.split(' ')
+
+    const event = [
+      'animationsend',
+      'webkitAnimationEnd',
+      'oAnimationEnd',
```

Comment:
thres a 4/5 line for loop taht can determine the users browser based on what styles are assigned to the `getComputed`/`style` object, then you only need to add events for those, means you dont have as many events running.